age/i18n/it-IT/age.ftl

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 Jack Grigg
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+### Localization for strings in the age library crate
+
+-age = age
+-rage = rage
+
+## CLI helpers
+
+cli-secret-input-required = È richiesto un input
+cli-secret-input-mismatch = Gli input non corrispondono
+
+cli-passphrase-desc = Digita la passphrase (lascia vuoto per generarne una sicura automaticamente)
+cli-passphrase-prompt = Passphrase
+cli-passphrase-confirm = Conferma la passphrase
+
+-flag-armor = -a/--armor
+-flag-output = -o/--output
+-output-stdout = -o -
+
+cli-truncated-tty = troncato; usa una pipe, una redirezione, o {-flag-output} per decrittare l'intero file
+
+err-detected-binary = rilevati dati non stampabili; rifiuto l'invio dell'output al terminale.
+rec-detected-binary = Puoi forzarlo con '{-output-stdout}'.
+
+err-deny-binary-output = rifiuto l'invio di output binario al terminale.
+rec-deny-binary-output = Intendevi usare {-flag-armor}? {rec-detected-binary}
+
+## Errors
+
+err-decryption-failed = Decrittazione fallita
```

Comment:
I think in general "decifrazione/cifrato/cifratura" is more common than "decrittazione/crittato/crittatura".

Reply:
Ok! Initially I wasn't totally convinced, but I'll stick with your suggestion. I'll change these here and there.